legacy/python/platform/src/actions/install_deps.rs

```diff
@@ -0,0 +1,129 @@
+use moon_action::Operation;
+use moon_console::{Checkpoint, Console};
+use moon_python_tool::PythonTool;
+use moon_utils::get_workspace_root;
+use std::path::Path;
+
+use crate::find_requirements_txt;
+
+pub async fn install_deps(
+    python: &PythonTool,
+    working_dir: &Path,
+    console: &Console,
+) -> miette::Result<Vec<Operation>> {
+    let mut operations = vec![];
+
+    // python.exec_python(args, working_dir)
+
+    if let Some(pip_config) = &python.config.pip {
```

Comment:
This may make more sense in the setup toolchain action.

Reply:
Some as above